samples/snippets/sample_data.proto

```diff
@@ -0,0 +1,37 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto2";
+
+message SampleData {
+  message SampleStruct {
+    optional int64 sub_int_col = 1;
+  }
+
+  optional bool bool_col = 1;
```

Comment:
Should we make one of the fields required?  It mucks up your sample code a bit, but it gets the coverage across nullable/required/repeated.

Reply:
A required "row number" field or something could be useful for testing. I forget if the wire format actually changes for required / optional. I thought it's just a client-side validation check.

Reply:
there's two variations:  if the proto is required, and schema is optional then you can get client side errors.  If the schema is required and proto is optional and unset, then you'll get an error from backend.